lldb/source/Host/common/Editline.cpp

```diff
@@ -393,7 +394,7 @@ void Editline::MoveCursor(CursorLocation from, CursorLocation to) {
   int fromLine = GetLineIndexForLocation(from, editline_cursor_row);
   int toLine = GetLineIndexForLocation(to, editline_cursor_row);
   if (toLine != fromLine) {
-    fprintf(m_output_file,
+    fprintf(m_output_stream_sp->GetFile().GetStream(),
```

Comment:
There are a lot of unguarded calls to `fprintf` and `fputs` in this file. Are there some guarantees about the state of the synchronized stream when these functions are executed? Or do we just have tons of unsynchronized writes to the output stream? :D